content/vault/v1.21.x/content/docs/license/pki.mdx

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+page_title: PKI-only clusters
+description: Technical overview of Vault licensing on PKI-only clusters
+---
+
+# PKI-only clusters
+
+Vault Enterprise 1.21 introduced the ability for customers to license a cluster based on the number of certificates issued through Vault's built-in PKI secrets engine. Customers choosing this option must deploy a cluster that is dedicated to PKI and operating in PKI-only mode.
```

Comment:
```suggestion
Vault Enterprise 1.21 introduced licensing for clusters based on the
number of certificates issued through the PKI secrets engine by deploying
clusters dedicated to PKI that operate in PKI-only mode.
```
Style correction: write in active voice, avoid possessives, avoid "this" as a pronoun